api/tools/src/main/java/org/pmiops/workbench/tools/ManageClusters.java

```diff
@@ -91,6 +101,42 @@ private static void listClusters(String apiUrl) throws IOException, ApiException
     System.out.println(String.format("listed %d clusters", count.get()));
   }
 
+  private static void describeCluster(
+      String apiUrl, String workbenchProjectId, String workbenchServiceAccount, String clusterId)
+      throws IOException, ApiException {
+    String[] parts = clusterId.split("/");
+    if (parts.length != 2) {
+      System.err.println(
+          String.format(
+              "given cluster ID '%s' is invalid, wanted format 'project/clusterName'", clusterId));
+      return;
+    }
+
+    // Leo's getCluster API swagger tends to be outdated; issue a raw getCluster request to ensure
+    // we get all available information for debugging.
+    ClusterApi client = newApiClient(apiUrl);
+    com.squareup.okhttp.Call call = client.getClusterCall(parts[0], parts[1], null, null);
```

Comment:
nit: I might name these arguments, even the null ones, for readability.

Reply:
Done for parts[0], parts[1]. For the nulls, I prefer to comment rather than having placeholder variables.